lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -6,49 +6,42 @@
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// const rearrangeCoefficient = require('./rearrangeCoefficient');
-// const reduceExponentByZero = require('./reduceExponentByZero');
+const rearrangeCoefficient = require('./rearrangeCoefficient');
+const reduceExponentByZero = require('./reduceExponentByZero');
 const reduceMultiplicationByZero = require('./reduceMultiplicationByZero');
-// const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
+const reduceZeroDividedByAnything = require('./reduceZeroDividedByAnything');
 const removeAdditionOfZero = require('./removeAdditionOfZero');
-// const removeDivisionByOne = require('./removeDivisionByOne');
```

Comment:
I'm in favor of deleting unused code instead of commenting it out. It's always in git if we need it.